chart/templates/_helpers.tpl

```diff
@@ -56,6 +56,13 @@ add below linux tolerations to workloads could be scheduled to those linux nodes
   operator: "Equal"
 {{- end -}}
 
+{{- define "node-tolerations-with-default" -}}
+{{ include "linux-node-tolerations" . | nindent 8 }}
+{{- with .Values.tolerations }}
+{{- toYaml . | nindent 8 }}
+{{- end -}}
```

Comment:
Instead of creating a helper function, let's directly insert this into the relevant areas.